File: Software Engineering/lab3/calculator.c
#include <stdio.h>

int add(int x, int y);
int multiply(int x, int y);
int division(int x, int y);
int subtraction(int x, int y);
int maximum(int x, int y);
int power(int x, int y);

void main() {
		 int number1, number2;
    printf("Welcome to my Calculator\n");
    printf("Enter 2 numbers\n");
    scanf("%d", &number1);
    scanf("%d", &number2);

    int sum = 0, product = 0, quotient = 0, substract = 0, max = 0;
    long long int pow = 0;

    sum = add(number1, number2);
    product = multiply(number1, number2);
    quotient = division(number1, number2);
    substract = subtraction(number1, number2);
    max = maximum(number1, number2);
    pow = power(number1, number2);

    printf("The sum of the 2 numbers is: %d \n", sum);
    printf("The product of the 2 numbers is: %d \n", product);
    printf("The quotient of the 2 numbers is: %d \n", quotient);
    printf("Number 1 - Number 2 is: %d \n", substract);
     printf("The maximum of the 2 numbers is: %d \n", max);
     printf("Number 1 to the power of Number 2 is: %d \n", pow);
}

File: Software Engineering/lab4/calculator.java
import java.util.Scanner;

public class calculator {
  public static void main(String [] args) {
  
    int number1, number2, quotient = 0, product = 0, sum = 0, sub = 0, maximum = 0;
    long numPow = 0;
    
    Scanner scan = new Scanner(System.in);

    System.out.println("Welcome to my digital calculator");
    System.out.println("Enter 2 numbers");
    number1 = scan.nextInt();
    number2 = scan.nextInt();
    
    division div = new division();
    multiply mult = new multiply();
    add plus = new add();
    substract minus = new substract();
    power pow = new power();
    max maxClass = new max();

    sum = plus.add(number1, number2);
    sub = minus.substract(number1, number2);
    product = mult.multiply(number1, number2);
    quotient = div.division(number1, number2);
    numPow = pow.numPower(number1, number2);
    maximum = maxClass.maxNum(number1, number2);

    System.out.println("The sum of the two numbers is: " + sum);
    System.out.println("Number1 - number2 is: " + sub);
    System.out.println("The product of the two numbers is: " + product);
    System.out.println("The quotient of the two numbers is: " + quotient);
    System.out.println("Number 1 to the power of number2 is: " + numPow);
    System.out.println("The max of the 2 numbers is: " + maximum);
  }
}
